bl/evaluation/evaluator.go

```diff
@@ -210,20 +276,33 @@ func (e *Evaluator) formatEvaluationResults(evaluationResults FailedRulesByFiles
 			totalPassedCount = totalPassedCount - 1
 		}
 
-		for ruleIdentifier, failedRuleData := range evaluationResults[filePath] {
+		for ruleIdentifier, failedRule := range evaluationResults[filePath] {
 			// file and rule not already exists in mapper
+			allConfigurationsAreSkipped := true
+
+			for _, configuration := range failedRule.Configurations {
+				if !configuration.IsSkipped {
+					allConfigurationsAreSkipped = false
+					break;
```

Comment:
To fix that linter run this in your project 

`gofmt -s -w ./bl/evaluation/evaluator.go`